ffi/Makefile.freebsd

```diff
@@ -1,6 +1,7 @@
 
 CXX = clang++ -std=c++11 -stdlib=libc++
-CXXFLAGS = $(LLVM_CXXFLAGS)
+CXXFLAGS := $(CPPFLAGS) $(CXXFLAGS) $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
+LDFLAGS := $(LDFLAGS) $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
 LDFLAGS = $(LLVM_LDFLAGS)
```

Comment:
Suggest removing this as its overwriting the line above.

Reply:
Will remove.